draft-foudil-securitytxt.md

```diff
@@ -732,8 +731,10 @@ The authors would like to acknowledge the help provided during the
 development of this document by Tom Hudson, Jobert Abma,
 Gerben Janssen van Doorn, Austin Heap, Stephane Bortzmeyer, Max Smith, Eduardo Vela and Krzysztof Kotowicz.
 
+The authors would also like to acknowledge the feedback provided by multiple members of IETF's
+LAST CALL, SAAG, SECDISPATCH lists.
```

Comment:
```suggestion
LAST CALL, SAAG, and SECDISPATCH lists.
```

Reply:
We should decide whether we use or don't use the Oxford comma and then be consistent (if we are for it, then the list of Contributors will need to have it -- or if we aren't, there should not be a comma here)